src/components/forms/validations.jsx

```diff
@@ -44,5 +45,5 @@ module.exports.validationHOCFactory = function (defaultValidationErrors) {
 };
 
 module.exports.defaultValidationHOC = module.exports.validationHOCFactory({
-    isDefaultRequiredValue: 'This field is required'
+    isDefaultRequiredValue: <intl.FormattedMessage id="teacherRegistration.validationRequired" />
```

Comment:
I'm fine with that – I was thinking we would just do that once it was actually used outside of `teacherregistration`, since it's not used anywhere else yet, but don't feel bound to that.